source-code/core2/src/fullExample.ts

```diff
@@ -96,11 +96,11 @@ message.body["en-US"] = {
 }
 
 inlang.messages.query.update({
-	where: { id: "myMessageId" },
+	id: "myMessageId",
 	data: message,
```

Comment:
why do we need a separate `data` prop?

Reply:
mixing `id` with `data` would likely become confusing, don't you think?

Reply:
```ts
inlang.messages.query.update({
	id: "myMessageId",
	body: [/* */],
})
```
?